viser/_message_api.py

```diff
@@ -70,6 +71,45 @@ def _colors_to_uint8(colors: onp.ndarray) -> onpt.NDArray[onp.uint8]:
 ]
 
 
+class TitlebarButton(dict):
+    def __init__(
+        self,
+        text: Optional[str] = None,
+        icon: Optional[str] = None,
+        href: Optional[str] = None,
+        variant: Optional[str] = None,
+    ):
+        super().__init__(self, text=text, icon=icon, href=href, variant=variant)
+
+
+class TitlebarImage(dict):
+    def __init__(self, imageSource: str, alt: str):
+        super().__init__(self, imageSource=imageSource, alt=alt)
+
+
+class TitlebarPadding(dict):
+    def __init__(self, width: str):
+        super().__init__(self, width=width)
+
+
+TitlebarObject: TypeAlias = Union[TitlebarButton, TitlebarImage, TitlebarPadding]
+
+
+class TitlebarConfig:
```

Comment:
(don't feel strongly) how do you feel about just having `titlebar_left`, `titlebar_center`, `titlebar_right`, as three separate inputs to `configure_theme()`?